2016/playbook/ACTIVITIES.md

```diff
@@ -27,8 +27,10 @@
 - [Free form projects](#free-form-projects)
   - [Brainstorming](#brainstorming)
   - [Building a prototype](#building-a-prototype)
+- [Hackathons](#hackathons)
 - [Using Slack](#using-slack)
   - [Asking for help](#asking-for-help)
+  - [Howdy](#howdy)
```

Comment:
Ah, gotcha, thanks!

On Thu, Jul 7, 2016 at 8:55 PM Max Wofford <notifications@github.com> wrote:

> In 2016/playbook/ACTIVITIES.md
> <https://github.com/hackclub/hackcamp/pull/732#discussion_r70022644>:
>
> >  - [Using Slack](#using-slack)
> >    - [Asking for help](#asking-for-help)
> > +  - [Howdy](#howdy)
>
> This just fills in the missing link to the section on Howdy.
>
> —
> You are receiving this because you were assigned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/hackcamp/pull/732/files/45d1e6075ffa108979b4d0e0a7b17b6f10737670#r70022644>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHYhMqb3JhXYkQ-k3LcTNoeYhSi5hYYnks5qTcongaJpZM4JHrXG>
> .
>